.github/workflows/test.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         run: |
           cd tests
           npm install
-          npx cypress run
+          npm run cypress:run
```

Comment:
NPX ignores our pinned version of cypress, I found it was installing cypress version 11. Using the NPM script will use our pinned version.